alicloud/service_alicloud_ecs.go

```diff
@@ -806,3 +806,27 @@ func (s *EcsService) DescribeSnapshotById(snapshotId string) (*ecs.Snapshot, err
 	}
 	return &resp.Snapshots.Snapshot[0], nil
 }
+
+func (s *EcsService) DescribeLaunchTemplate(id string) (set ecs.LaunchTemplateVersionSet, err error) {
+	req := ecs.CreateDescribeLaunchTemplateVersionsRequest()
+	req.LaunchTemplateId = id
+
+	raw, err := s.client.WithEcsClient(func(ecsClient *ecs.Client) (interface{}, error) {
+		return ecsClient.DescribeLaunchTemplateVersions(req)
+	})
+	if err != nil {
+		if IsExceptedError(err, "InvalidLaunchTemplate.NotFound") {
+			err = WrapErrorf(err, NotFoundMsg, ProviderERROR)
+			return
+		}
+		err = WrapErrorf(err, DefaultErrorMsg, id, req.GetActionName(), AlibabaCloudSdkGoERROR)
+		return
+	}
+	resp := raw.(*ecs.DescribeLaunchTemplateVersionsResponse)
+	if len(resp.LaunchTemplateVersionSets.LaunchTemplateVersionSet) == 0 {
+		err = WrapErrorf(GetNotFoundErrorFromString("LaunchTemplate"), NotFoundMsg, AlibabaCloudSdkGoERROR)
```

Comment:
I am sorry. The line 827 should be `WrapErrorf(Error((GetNotFoundMessage("LaunchTemplate", id)), NotFoundMsg, ProviderError)
 `